suites/load-testing/fence/presigned-url.js

```diff
@@ -0,0 +1,32 @@
+import { check } from "k6";
+import http from "k6/http";
+
+const host = "qa-brain.planx-pla.net";
+const guid = "00037250-f2e5-47e2-863c-5f225c7f79e8";
+const token = open('../token.txt', 'utf8').trim();
```

Comment:
I wonder if we could leverage the `getAccessToken.js` here by asking the user to place a `credentials.json` somewhere...

Just a thought though. No blockers.

Reply:
👍